packages/wallet/backend/src/transaction/service.ts

```diff
@@ -87,6 +87,7 @@ export class TransactionService implements ITransactionService {
       )
       .fullOuterJoinRelated('[walletAddress, account.user]')
       .where('account:user.id', userId)
+      .whereNot('status', 'DELETED')
       .whereNotNull('transactions.id')
```

Comment:
as id is the primary key, curious why we have this check here
I think it should be deleted